test/services/permissions.test.js

```diff
@@ -815,26 +815,99 @@ describe('services > permissions', () => {
         });
 
         describe('check if it requests permissions after a denied access', () => {
-          it('should return a resolved promise', async () => {
-            expect.assertions(1);
+          describe('with the "browseEnabled" permission', () => {
+            it('should return a reject promise first and then a resolve promise', async () => {
+              expect.assertions(2);
 
-            nock.cleanAll();
-            nockObj.get('/liana/v3/permissions?renderingId=1')
-              .reply(200, {
-                meta: { rolesACLActivated: true },
-                data: {
-                  collections: {
-                    Users: {
-                      collection: {
-                        browseEnabled: true,
+              PermissionsGetter.cleanCache();
+              nock.cleanAll();
+              nockObj.persist().get('/liana/v3/permissions?renderingId=1')
+                .reply(200, {
+                  meta: { rolesACLActivated: true },
+                  data: {
+                    collections: {
+                      Users: {
+                        collection: {
+                          browseEnabled: false,
+                        },
                       },
                     },
+                    renderings: {
+                      1: {},
+                    },
                   },
-                },
-              });
+                });
 
-            await expect(new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'browseEnabled', { userId: 1 }))
-              .toResolve();
+              await expect(new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'browseEnabled', { userId: 1 }))
+                .rejects.toThrow("'browseEnabled' access forbidden on Users");
+
+              nockObj.persist(false);
```

Comment:
I accepted for the first PR but I won;'t accept on this one. Please do Unit test !